Source/Core/VideoCommon/BPStructs.cpp

```diff
@@ -285,6 +285,8 @@ static void BPWritten(const BPCmd& bp)
     if (g_bRecordFifoData)
       FifoRecorder::GetInstance().UseMemory(addr, tlutXferCount, MemoryUpdate::TMEM);
 
+    TextureCacheBase::InvalidateBindPoint(0xff);
```

Comment:
Can we replace this with an `InvalidateAllBindPoints()` function or something.

If we are going to do invalidation of individual stages, we will need more precise functions than this one which takes a bitmask. No point exposing the internals when they aren't even used.